package.json

```diff
@@ -74,14 +75,14 @@
     "@types/marked": "^5.0.0",
     "@types/natural-compare": "^1.4.1",
     "@types/ncp": "^2.0.5",
-    "@types/node": "^20.0.0",
+    "@types/node": "18.16.9",
```

Comment:
Is this downgrade necessary? Merge conflict?

Reply:
Probably an automated migration that was too eager, I’ll revert